### PR TITLE
fix(#51): reset the cursor position of stream

### PR DESCRIPTION
**Pull request by SalehDehqanpour**
fix(#51): reset the cursor position of stream so be idempotent and transparent to users

### CODE REVIEW

File: filetype/utils.py [hunk 1]
```diff
@@ -57,7 +59,13 @@ def get_bytes(obj):
         TypeError: if obj is not a supported type.
     """
     try:
-        obj = obj.read(_NUM_SIGNATURE_BYTES)
+        if type(obj) is IOBase:
```
Comment (dosas) [inline filetype/utils.py:62]: @salehdeh76 Could you use ```if isinstance(obj, IOBase)```

**Comment by salehdeh76** [comment]
@dosas I fixed it. Please review it again. Thanks

**Comment by dosas** [comment]
@salehdeh76 Did you see this PR https://github.com/h2non/filetype.py/pull/51/files ?

There is already an open question?

Also, a unit test is always a plus :D

**Comment by guigarfr** [comment]
https://github.com/h2non/filetype.py/pull/51 was already fixed and merged, this PR can be closed @h2non @salehdeh76

**Comment by guigarfr** [comment]
This problem was solved in https://github.com/h2non/filetype.py/pull/137 and merged. You can close the PR @h2non @salehdeh76

**Pull request closed without merging**